packages/models/src/errors.ts

```diff
@@ -340,6 +350,15 @@ export function attributeNotFound(attributeId: string): AttributeProcessError {
   );
 }
 
+export function attributeDuplicate(
+  attributeNameSeed: string
+): AttributeProcessError {
+  return new AttributeProcessError(
+    `Error during Attribute creation with name ${attributeNameSeed}`,
```

Comment:
Why not give a more informative error?
```suggestion
    Attribute ${attributeNameSeed} already exists,
```

Reply:
Actually the other parameter `ErrorTypes.DuplicateEserviceName` already contains an explanation of the error.

Reply:
You

> Actually the other parameter `ErrorTypes.DuplicateEserviceName` already contains an explanation of the error.
You're right, but we should create a `DuplicateAttributeName`. There's a PR that will refactor the errors.

Reply:
Sorry, I didn't even notice while I was typing. Yes there is `DuplicateAttributeName` implemented here.